build fails in macos due to `failed to resolve: could not find open_xr_action in engine`

I updated from 4.1.1 to 4.1.3, and gdext from `00ba8c9d` to `02f4acf5` and I started getting the following error in MacOS (Windows and Linux are OK):

```
error[E0433]: failed to resolve: could not find `open_xr_action` in `engine`
  --> /Users/mateomiccino/.cargo/git/checkouts/gdext-76630c89719e160c/02f4acf/godot-core/src/gen/classes/open_xr_interaction_profile_metadata.rs:88:223
   |
88 | ...dotString, action_type: crate::engine::open_xr_action::ActionType,) {
   |                                           ^^^^^^^^^^^^^^ could not find `open_xr_action` in `engine`

error[E0433]: failed to resolve: could not find `open_xr_action` in `engine`
  --> /Users/mateomiccino/.cargo/git/checkouts/gdext-76630c89719e160c/02f4acf/godot-core/src/gen/classes/open_xr_interaction_profile_metadata.rs:90:113
   |
90 |             type CallSig = ((), GodotString, GodotString, GodotString, GodotString, GodotString, crate::engine::open_xr_action::ActionType);
   |                                                                                                                 ^^^^^^^^^^^^^^ could not find `open_xr_action` in `engine`
```

I compared the generated files (at .cargo/git/checkouts/gdext-76630c89719e160c/02f4acf/godot-core/src/gen/classes) between Linux and Mac, and Mac is not generating the `open_xr_**` files, it just generating the `open_xr_interaction_profile_metadata`.

More details in a GH Action:
https://github.com/decentraland/godot-explorer/actions/runs/6824669204/job/18561040610

I'm using pre-build
```
[patch."https://github.com/godot-rust/godot4-prebuilt".godot4-prebuilt]
git = "https://github.com//godot-rust/godot4-prebuilt"
branch = "4.1.3"
```
entire cargo.toml: https://github.com/decentraland/godot-explorer/blob/main/rust/decentraland-godot-lib/Cargo.toml

--- a/.github/workflows/update-docs.yml
+++ b/.github/workflows/update-docs.yml
@@ -58,7 +58,9 @@ jobs:
       # Opened/reopened/updated PR: include PR author + title
       - name: "Construct JSON (for PR sync)"
         if: github.event_name == 'pull_request' && github.event.action != 'closed'
+        # Escape double quotes in PR title, as it will be used in a JSON string.
         run: |
+          escapedPrTitle=$(echo "${{ github.event.pull_request.title }}" | sed 's/"/\\"/g')
           payload=$(cat <<'HEREDOC'
           {
             "op": "put",
@@ -68,7 +70,7 @@ jobs:
             "commit-sha": "${{ github.event.pull_request.head.sha }}",
             "date": "${{ github.event.pull_request.updated_at }}",
             "pr-author": "${{ github.event.pull_request.user.login }}",
-            "pr-title": "${{ github.event.pull_request.title }}"
+            "pr-title": "$escapedPrTitle"
           }
           HEREDOC)
           echo "VAR=$payload"

--- a/godot-codegen/src/special_cases.rs
+++ b/godot-codegen/src/special_cases.rs
@@ -53,18 +53,10 @@ pub(crate) fn is_class_deleted(class_name: &TyName) -> bool {
 
     // OpenXR has not been available for macOS before 4.2.
     // See e.g. https://github.com/GodotVR/godot-xr-tools/issues/479.
+    // Do not hardcode a list of OpenXR classes, as more may be added in future Godot versions; instead use prefix.
     #[cfg(all(before_api = "4.2", target_os = "macos"))]
-    match class_name {
-        | "OpenXRHand"
-        | "OpenXRAction"
-        | "OpenXRActionMap"
-        | "OpenXRActionSet"
-        | "OpenXRInteractionProfile"
-        | "OpenXRIPBinding"
-        | "OpenXRInterface"
-
-          => return true,
-        _ => {}
+    if class_name.starts_with("OpenXR") {
+        return true;
     }
 
     // ThemeDB was previously loaded lazily

